### PR TITLE
Polaris lorepatches

### LINKED ISSUE #32: People crumble to dust at the ripe age of 70

### LINKED ISSUE #34: schechi doesn't have a shorthand
image

### PR DESCRIPTION
**Pull request by Atermonera**
## Description of changes
Porting Cerebulon's patches & tweaks to add our lore background, in a more atomic/upstream-compatible manner.
Currently includes
#1 #3 #5 
Fixes #34, fixes #32
Still need to do:
- [x] Find/replace defines
- [x] ~"Location" -> "Home Culture" -- Really these should be a many-many mapping between places and the cultures present there, still thinking about how to approach~ Keep locations for lore events, remove from char setup, 
- [x] Add Citizenship decls and add to char setup
- [x] Polaris modpack
  - [x] Languages
  - [x] Species - Weighing whether to copy neocorvid/bayliens & modify locally or just overwrite the modpacks
  - [x] Culture decls
  - [x] "Sol Central" -> "Sol Confederate Gov"
- [x] Species lore updates

Char setup now looks like
this

Added datums for e.g., taj:
this
## Authorship
@Atermonera 
@Cerebulon

## Changelog
:cl:
add: Create Polaris modpack.
add: Add Polaris-specific cultures.
add: Add Polaris-specific citizenships.
del: Remove "Homeworld/Residence" from char setup, since everybody lives in one of a couple locations.
/:cl:

### CODE REVIEW

**Comment by Atermonera** [comment]
New approach, I think this should cover our usecase and not be a huge PITA to send upstream.
Character setup on ministation looks like:
image

And the rest of the modpacks should at least _compile_, though I was admittedly barebones in my descriptions.

File: code/__defines/culture.dm [hunk 1]
```diff
@@ -2,10 +2,12 @@
 #define TAG_HOMEWORLD "home_system"
 #define TAG_FACTION   "faction"
 #define TAG_RELIGION  "religion"
+#define TAG_CITIZENSHIP "citizenship"
 
 var/global/list/ALL_CULTURAL_TAGS = list(
 	TAG_CULTURE =   "Culture",
-	TAG_HOMEWORLD = "Residence",
+//	TAG_HOMEWORLD = "Residence",
+	TAG_CITIZENSHIP = "Citizenship",
```
Comment (Atermonera) [inline code/__defines/culture.dm:10]: Literally, "Remove residence from character setup and add citizenship", basically.

File: maps/~mapsystem/maps.dm [hunk 2]
```diff
@@ -128,17 +128,19 @@ var/global/const/MAP_HAS_RANK = 2		//Rank system, also togglable
...
 	var/list/available_cultural_info = list(
-		TAG_HOMEWORLD = list(/decl/cultural_info/location/other),
-		TAG_FACTION =   list(/decl/cultural_info/faction/other),
-		TAG_CULTURE =   list(/decl/cultural_info/culture/other),
-		TAG_RELIGION =  list(/decl/cultural_info/religion/other)
+		TAG_HOMEWORLD =   list(/decl/cultural_info/location/other),
+		TAG_FACTION =     list(/decl/cultural_info/faction/other),
+		TAG_CULTURE =     list(/decl/cultural_info/culture/other),
+		TAG_RELIGION =    list(/decl/cultural_info/religion/other),
+		TAG_CITIZENSHIP = list(/decl/cultural_info/citizenship/other)
```
Comment (Atermonera) [inline maps/~mapsystem/maps.dm:135]: As with the other PR, we ought to set this on a specific map, but doing it here creates blanket baseline so most other maps should compile w/ the changed ALL_CULTURAL_DATUMS list

File: mods/species/bayliens/_bayliens.dme [hunk 3]
```diff
@@ -15,12 +15,12 @@
 #include "skrell\_skrell.dm"
 #include "skrell\datum\accessory.dm"
 #include "skrell\datum\blood.dm"
+#include "skrell\datum\citizenship.dm"
 #include "skrell\datum\culture.dm"
 #include "skrell\datum\descriptor.dm"
 #include "skrell\datum\emotes.dm"
 #include "skrell\datum\faction.dm"
 #include "skrell\datum\language.dm"
-#include "skrell\datum\location.dm"
```
Comment (Atermonera) [inline mods/species/bayliens/_bayliens.dme:23]: We might eventually want to add this back to get codex entries & ATC fluff for actual skrell places but the things that were in here are closer to citizenship than to places anyways

File: mods/species/neoavians/datum/language.dm [hunk 4]
```diff
@@ -1,41 +1,14 @@
...
-/decl/language/corvid/get_random_name(gender)
-	. = capitalize((gender == FEMALE) ? pick(global.using_map.first_names_female) : pick(global.using_map.first_names_male))
-	. += " [pick(list("Albus","Corax","Corone","Meeki","Insularis","Orru","Sinaloae", "Enca", "Edithae", "Kubaryi"))]"
-	. += " [pick(list("Hyperion","Earth","Mars","Venus","Neith","Luna","Halo","Pandora","Neptune","Triton", "Haumea", "Eris", "Makemake"))]"
-
 /decl/language/neoavian
-	name = "Neo-Avian Pidgin" // pigeon
-	desc = "A cluster of melodic, trilling languages spoken by the majority of neo-avian subcultures."
+	name = "Schechi"
+	desc = "A trilling language spoken by the diminutive Teshari."
```
[the remark this answers is not in the document]
Comment (Cerebulon) [reply mods/species/neoavians/datum/language.dm:3]: Akhani is also missing this (my mistake from original PR)

[the remark this answers is not in the document]
Comment (Atermonera) [reply mods/species/neoavians/datum/language.dm:3]: Might still fiddle with them bc 2 capital letters feels janky

File: maps/~mapsystem/maps.dm (hunk 2 above)
Comment (Atermonera) [reply maps/~mapsystem/maps.dm]: Updated to only affect tradeship, will need to copy that to other maps as needed

**Pull request merged**